src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
+type Color = 'red' | 'green' | 'blue';
```

Comment:
Using an enum is better here. It's more flexible when you want to add new values.